src/impls.rs

```diff
@@ -15,21 +15,71 @@
 // along with Tellor. If not, see <http://www.gnu.org/licenses/>.
 
 use super::*;
-use sp_runtime::traits::Hash;
+use crate::constants::DECIMALS;
+use frame_support::traits::fungible::Inspect;
+use sp_runtime::{
+	traits::{CheckedAdd, CheckedMul, Hash},
+	ArithmeticError,
+};
 
 impl<T: Config> Pallet<T> {
+	/// Funds the staking account with staking rewards (paid by autopay and minting)
+	/// # Arguments
+	/// * `amount` - The amount of tokens to fund the staking account with.
 	pub(super) fn add_staking_rewards(amount: BalanceOf<T>) -> DispatchResult {
```

Comment:
1. Does pub(super) mean this function cannot be called by anyone? If not, why not? I think the original one lets anyone call it. See [here](https://github.com/tellor-io/tellorFlex/blob/main/contracts/TellorFlex.sol#L134)
2. I'd like to go over on the call tomorrow, or if you could explain a bit here about the `tips()` account that the funds are being transferred from, so I can understand the transfer a bit better.

Reply:
1. Will get back to you on the call / will ask Tim and the others, because I'm not sure.
2. And only the pallet owner has control of this `tips` sub-account? Thanks for clarifying.

Reply:
1. "Anyone should be able to call it and incentivize reporters. Not everyone wants to just give away money but people could use it to subsidize usage and bootstrap the network."

Reply:
Added in latest commit.